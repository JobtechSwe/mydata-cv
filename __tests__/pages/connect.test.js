/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import ConnectPage from '../../pages/connect.js'

describe('pages/ConnectPage', () => {
  it('renders', () => {
    const page = shallow(<ConnectPage />)

    expect(page.find('ConnectForm').exists()).toEqual(true)
  })
})

describe('With Snapshot Testing', () => {
  it('renders', () => {
    const component = renderer.create(<ConnectPage />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
