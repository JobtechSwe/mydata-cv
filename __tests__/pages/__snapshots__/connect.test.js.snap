// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing renders 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <label>
      Name:
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <input
      id="submit"
      type="submit"
      value="Submit"
    />
  </form>
  <div>
    Connecting...
  </div>
  <div>
    Done
  </div>
  <div>
    Error
  </div>
</div>
`;
